eval/src/vespa/eval/eval/interpreted_function.cpp

```diff
@@ -216,8 +211,25 @@ struct ProgramBuilder : public NodeVisitor, public NodeTraverser {
 
     //-------------------------------------------------------------------------
 
+    void make_const_op(const Node &node, const Value &value) {
```

Comment:
looks like we can just remove the "node" parameter to make_const_op()

Reply:
we need to type to do more fancy stuff in the future...